### DISCUSSION TITLE
RNN Modal vs. RN Modal Event Handling

### DISCUSSION THREAD

**Discussion by chris-cornell** [wix/react-native-navigation#7400] [Q&A]
when i dismiss a react-native `<Modal />` on iOS via swipe gesture, should RNN be calling [sendModalAttemptedToDismissEvent from RNNEventEmitter](https://github.com/wix/react-native-navigation/blob/master/lib/ios/RNNEventEmitter.m#L133)?

i'm getting a crash because the above handler is firing but it's _missing a componentId_ - from my understanding, it doesn't have one since it's not registered w/ react-native-navigation, but a declarative Modal that's a child of the `RNN` view

as far as i can understand, RN modals should trigger this [react-native iOS event](https://github.com/facebook/react-native/commit/c29ec46b0eee99670ce7762898fe3a4810db968b).

if anyone has any insight, it would be greatly appreciated. i suspect something in my setup / RNN<>iOS link is incorrect.

**Comment by donatoaguirre24** [comment]
We are having the same issue, but in our case the `Modal` is used internally by a third party dependency. Were you able to patch or workaround this issue while keep using RN's modal?
